<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>xseregax</author>
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>906</width>
    <height>766</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::NoContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>srgUartTools</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <widget class="QGroupBox" name="gbPort">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>291</width>
      <height>131</height>
     </rect>
    </property>
    <property name="title">
     <string>Com Port</string>
    </property>
    <widget class="QComboBox" name="cbPorts">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>141</width>
       <height>23</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string/>
      </property>
     </item>
    </widget>
    <widget class="QPushButton" name="pbPortOpen">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>30</y>
       <width>89</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Open</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cbSpeeds">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>141</width>
       <height>23</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>3</number>
     </property>
     <item>
      <property name="text">
       <string>9600</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>19200</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>38400</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>57600</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>115200</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="lbPortStatus">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>53</width>
       <height>14</height>
      </rect>
     </property>
     <property name="text">
      <string>status</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="gbCommands">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>0</y>
      <width>581</width>
      <height>131</height>
     </rect>
    </property>
    <property name="title">
     <string>Commands</string>
    </property>
    <widget class="QLineEdit" name="leCmd">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>181</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pbCmd">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>30</y>
       <width>89</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>send</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="gbOutput">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>160</y>
      <width>881</width>
      <height>561</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="title">
     <string>Output</string>
    </property>
    <widget class="QTabWidget" name="tabOutput">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>861</width>
       <height>541</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Logs</string>
      </attribute>
      <widget class="QTextEdit" name="teOutput">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>0</y>
         <width>831</width>
         <height>501</height>
        </rect>
       </property>
       <property name="acceptDrops">
        <bool>false</bool>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Graphics</string>
      </attribute>
      <widget class="QwtPlot" name="qwtPlot">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>781</width>
         <height>461</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="btnClear">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>480</y>
         <width>89</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Clear</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>906</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QwtPlot</class>
   <extends>QFrame</extends>
   <header>qwt_plot.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
